Yggtunnel config names/variables (#379)
tomeshnet/prototype-cjdns-pi: docs/MODULES.md

### Additional configuration
Additional configurations can be made in the file `/etc/yggdrasil.iptunnel.conf`

Section `[general]`

**yggdrasil-interface**
*Server Only*

Defines the yggdrasil interface.

Default: ygg0

> Value: `Yggdrasill interface on the system. Usually ygg0 or tun0`

Reviewer: ```suggestion
Value: `Yggdrasil interface on the system. Usually ygg0 or tun0.`
```

**exit-interface**
*Server Only*

Defines the exit interface that is connected to the internet.

Default: *eth0*
Value: `Any interface on the system`

Section `[ipv6tunnel]`

**nat**

> Disables masquerading of IPv6 tunnels. Set to `false` when routable addresses are being used across the tunnel. This is used when you route another ipv6 route to be used by the remote node to issue IPv6 addresses to it's clients.

Reviewer: ```suggestion
Disables masquerading of IPv6 tunnels. Set to `false` when routable addresses are being used across the tunnel. This is used when you route another IPv6 route to be used by the remote node to issue IPv6 addresses to its clients.
```

Default: *true*
Values: `true` , `false`

> To add advertising of this subnet, add a prefix to `/etc/radvd.conf` under the correct interface.  For example if the subnet you are routing is prefix `20xx:xxxx:xxxx::/64`

Reviewer: ```suggestion
To add advertising of this subnet, add a prefix to `/etc/radvd.conf` under the correct interface.  For example, if the subnet you are routing has this prefix `20xx:xxxx:xxxx::/64`, you could write this:
```

> Example of /etc/radvd.conf file

Reviewer: ```suggestion
```

```
interface wlan-ap {
  AdvSendAdvert on;
  MaxRtrAdvInterval 30;
  prefix 20xx:xxxx:xxxx::/64
  {
    AdvOnLink on;
    AdvAutonomous on;
  }
};
```

If you installed hostapd you will have this already configured with a prefix.  Simply add the new prefix under the existing one.

**subnet**
*Server only*

> Defines ipv6 addresses to add to route table that will be routed over through yggdrasil. Must match IPs in `yggdrasil.iptunnel.server`

Reviewer: ```suggestion
Defines IPv6 addresses to add to the route table that will be routed over through Yggdrasil. Must match IPs in `yggdrasil.iptunnel.server`
```

Default: *fd00::/64*
Value: `Any ipv6 address range`

**subnet-interface**
*Client Only*

Defines the interface that will be used to route the routed IPv6 subnet.

> Default: wlan-ap

Reviewer: ```suggestion
Default: *wlan-ap* <-- or `wlan-ap` under my suggestion -->
```

Value: `Any interface on the system`

Section `[iptunnel]`

**subnet**
*Server only*

> Defines ip addresses to add to route table that will be routed over through yggdrasil. Must match ips in `yggdrasil.iptunnel.server`

Reviewer: ```suggestion
Defines IP addresses to add to the route table that will be routed over through Yggdrasil. Must match the IPs in `yggdrasil.iptunnel.server`
```

Default: *10.10.0.0/24*

> Value: `Any ipv4 address range`

Reviewer: ```suggestion
Value: `Any IPv4 address range`
```
